src/components/victory-candlestick/helper-methods.js

```diff
@@ -110,18 +110,21 @@ export default {
     }
   },
 
-  getDataStyles(datum, style, props) {
+  getDataStyles(datum, style, props) { // eslint-disable-line complexity
     const stylesFromData = omit(datum, [
       "x", "y", "size", "name", "label", "open", "close", "high", "low"
     ]);
     const fillCheck = datum.fill || style.fill;
     const strokeCheck = datum.stroke || style.stroke;
-    const strokeColor = fillCheck && !strokeCheck ? fillCheck
-    : strokeCheck;
     const candleColor = datum.open > datum.close ?
             props.candleColors.negative : props.candleColors.positive;
+    const transparentCheck = datum.stroke === "transparent"
```

Comment:
Can you extract some of this logic into a function like `isInvisible(attr)` and then your transparentCheck could just be:
```
const transparenCheck = isInvisible(datum.stroke) || isInvisible(style.stroke);
```